config/database.yml

```diff
@@ -1,19 +1,20 @@
-base: &base
-  encoding: utf8
-  url: <%= ENV.fetch("DATABASE_URL", "trilogy://root@localhost") %>
+default: &default
+  adapter: trilogy
+  encoding: utf8mb4           # <-- forces UTF-8 everywhere
+  collation: utf8mb4_unicode_ci  # <-- portable across MySQL 5.7/8.0 and MariaDB
+  pool: <%= ENV.fetch("RAILS_MAX_THREADS", 5) %>
```

Comment:
I actually get slightly nervous about making the pool the same size as the rails thread count ... mostly as it's possible that the rails application logic can (potentially) throw things out to their _own_ background threads ... 

I'm not really sure just _how_ much this matters, but it's own of those things that makes me :hmmm: